dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/tests/Audacia.CodeAnalysis.Analyzers.Test/Rules/MagicNumberAnalyzerTests.cs

```diff
@@ -312,22 +313,279 @@ private int Calculate(int? arg)
         }
 
         [TestMethod]
-        public void No_Diagnostic_For_Variable_With_Magic_Number_Assignment_If_The_Magic_Number_Is_A_Multiple_Of_10()
+        public void Diagnostic_For_For_Loop_Magic_Number()
```

Comment:
This would read better if it included `With`, and would make it consistent with the other tests in this class i.e
`Diagnostic_For_For_Loop_With_Magic_Number`

Reply:
a few of this tests should say `Diagnostic_For` rather than just `Diagnostic` if that's OK